tests/std/tests/GH_003570_allocate_at_least/test.cpp

```diff
@@ -72,10 +72,15 @@ void test_container() {
 }
 
 void test_deque() {
-    deque<int, signalling_allocator<int>> d;
-    d.resize(100);
+    deque<size_t, signalling_allocator<size_t>> d;
+    for (size_t i = 0; i < 100; i++) {

```

Comment:
Style: We conventionally prefer preincrement, using postincrement only when it's necessary. Occurs below.